docs/security/trust/deploying_notary.md

```diff
@@ -0,0 +1,37 @@
+<!--[metadata]>
++++
+title = "Deploying Notary"
+description = "Deploying Notary"
+keywords = ["trust, security, notary, deployment"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Deploying Notary
+
+## Using Docker Compose
```

Comment:
This is called stacking your heads. 
![image](https://cloud.githubusercontent.com/assets/1347209/11100288/2bf166fc-8865-11e5-93d8-12eded3bac51.png)